### DISCUSSION TITLE
Looking for help with recovering SnipeIT

### DISCUSSION THREAD

**Discussion by cwsluke1** [snipe/snipe-it#10195] [Q&A]
Hello, I have a windows server that got updated to a new version and broke IIS. I tried to fix it but could not. I ended up reloading a fresh install of SnipeIT and tried copying the database files and uploads to the new server but they broke the install. I'm not sure where to post but I need help fixing this. I already dumped hours into resolving this...Thank you

**Comment by snipe** [comment]
Hi - we have plenty of IIS users who use Snipe-IT and upgrade all the time, so I doubt very much that it's something we did. But even if it was, without more information, we're not going to be able to help you. What's in your error logs, what are you seeing, etc etc? Screenshots? From what version of Snipe-IT *to* what version of Snipe-IT? Literally any info here would help. 

Also you almost never want to copy over database *files*. You'd want to restore from backup. 

Please see the Getting Help documentation.
https://snipe-it.readme.io/docs/getting-help

**Reply by cwsluke1** [reply]
Hello, thank you for the response. I am sorry I did not clarify. I had upgraded the Server from windows 2012 to 2012 R2 and did not make a config backup of IIS. I tried to rebuild IIS to the specs required for Snipe but it would continue to cause errors. so I removed the role from the server. After that I could not add IIS back as the install failed. I still have the original directories for Snipe on the server, but I am afraid I do not have a backup file.

I am very new to this and I have read the original help docs as well as other forums and sites. If we were able to move hosted would you be able to recover the old database/assets? Thank you for your time

**Comment by snipe** [comment]
Do you have backup files from the admin > backups utility? (Should be a .zip that looks something like snipe-it-yyyy-mm-dd.zip)

You may need to re-add your web.config in IIS (it’s in the public root)

**Reply by cwsluke1** [reply]
Unfortunately, I do not have any backup zip files. Should my focus be on recovering the current (broken) server? Or is it possible to migrate the database/settings to a new build?

**Comment by snipe** [comment]
That’s unfortunate, and why we recommend scheduling backups via the task scheduler. 

Can you still access the server at all? Is it only the front-end (via web browser) that’s busted? If so, what’s the error. Once again, we really need more info to help you.

**Reply by cwsluke1** [reply]
I have access to the server. I can see the snipe directories and Marana DB database files. The issue was with IIS so I uninstalled IIS and I think I made it worse. I cannot reinstall IIS through windows server manager for some reason, therefore I can't even troubleshoot snipe. 

I understand all the issues with the server such as IIS is not related to snipe, I just wanted to know if there was a way to recover the database files without a backup file or snipe being operational. Then I could theoretically transfer them to a new install of snipe.

**Reply by cwsluke1** [reply]
Is this a lost cause? If we could recover the assets for a hosted solution I could convince my manager to move us hosted.
